webapp/components/autosize_textarea.jsx

```diff
@@ -84,7 +84,9 @@ export default class AutosizeTextarea extends React.Component {
                     <textarea
                         ref='reference'
                         style={{height: 'auto', width: '100%'}}
-                        value={value || placeholder}
+                        disabled={true}
```

Comment:
This is the hidden textbox that only gets used for dimension calculation. Disabling it prevents input from being added to it and prevents you from tabbing to it as well